### PR TITLE
Add post on GitHub pro tips

### PR DESCRIPTION
**Pull request by mriddle**
Sharing some pro tips that some of us might not know. 
Used for a lunch and learn in the US office

### CODE REVIEW

**Comment by vinbarnes** [comment]
[rendered markdown](https://github.com/lonelyplanet/blog/blob/mr-github-pro-tips/_posts/2014-06-17-GitHub-Tips-n-Tricks.md)

File: _posts/2014-06-17-GitHub-Tips-n-Tricks.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+[`sloth in:file -extension:rb @lonelyplanet`](https://github.com/search?q=sloth+in%3Afile+-extension%3Arb+%40lonelyplanet&type=Code&ref=searchresults)
+
+## Cross-references
+
+Shipping something dependant on another repo
+
+`git commit -m "Roll out new UI from lonelyplanet/rizzo#782"`
+
+Late commit relating to a merged PR (will show up in the discussion)
```
Comment (leejones) [inline _posts/2014-06-17-GitHub-Tips-n-Tricks.md:57]: @mriddle what did you mean by “Late commit”?

**Comment by vinbarnes** [comment]
@mriddle made some tweaks to get the look/feel more consistent, etc. If I took editorial liberties that aren't in keeping with your vision for the article feel free to revert.

**Comment by vinbarnes** [comment]
Minor preference for having or own slothy reply gif rather than using GH's. :grin:

File: _posts/2014-06-17-GitHub-Tips-n-Tricks.md (hunk 1 above)
Comment (mriddle) [inline _posts/2014-06-17-GitHub-Tips-n-Tricks.md:57]: @leejones Committing code that's relevant to a pull request which has already been merged.

**Comment by vinbarnes** [comment]
5810a77 :zap:

File: _posts/2014-06-17-GitHub-Tips-n-Tricks.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+[`rspec path:Gemfile @lonelyplanet -repo:lonelyplanet/openresty-statsd`](https://github.com/search?q=rspec+path%3AGemfile+%40lonelyplanet+-repo%3Alonelyplanet%2Fopenresty-statsd&type=Code&ref=searchresults)
+
+**Find all files that contain sloths within your organization**
+
+[`sloth in:file -extension:rb @lonelyplanet`](https://github.com/search?q=sloth+in%3Afile+-extension%3Arb+%40lonelyplanet&type=Code&ref=searchresults)
+
+**Find sensitive information**
+
+* [`"PRIVATE KEY" in:file path:.ssh/id_rsa`](https://github.com/search?q=%22PRIVATE+KEY%22+in%3Afile+path%3A.ssh%2Fid_rsa&type=Code&ref=searchresults)
```
Comment (TimMoore) [inline _posts/2014-06-17-GitHub-Tips-n-Tricks.md]: I know it's already public, but I'm not so sure it's good form to "out" people this way.

Though, https://github.com/johndouthat/mydotfiles/blob/81b657398c71c33c6b7459c5aa8c2f070adadca9/.ssh/id_rsa I LOL'ed

File: _posts/2014-06-17-GitHub-Tips-n-Tricks.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+Late commit relating to a merged PR (will show up in the discussion)
+
+`git commit -m "(#1420) Fixes broken sloth animation"`
+
+The same thing can also be done from GitHub comments
+
+## Code comparison
+
+You may already be familiar with using [http://github.com/<org>/<repo>/compare]() to create a pull-request from a branch, but it's also super useful for filtering commits.
```
Comment (TimMoore) [inline _posts/2014-06-17-GitHub-Tips-n-Tricks.md]: I think you need to escape `<org>` and `<repo>` here. At least in the rendered view of the PR, they're invisible.

Putting backticks around it should work.

**Comment by TimMoore** [comment]
Nice post! I tweaked a couple of things, too, and to echo Kevin: feel free to revert if you don't like any of them (although I think I just changed his changes).

File: _posts/2014-06-17-GitHub-Tips-n-Tricks.md (hunk 2 above)
Comment (mriddle) [inline _posts/2014-06-17-GitHub-Tips-n-Tricks.md]: Great point, I was trying to convey how easy it is and that other GitHub users should be aware.
I'm going to add the user param so just that file shows up instead, also made me laugh.

**Comment by leejones** [comment]
:+1: nice work @mriddle :sparkles:

**Comment by mriddle** [comment]
Thanks for the feedback team! :sparkling_heart:

**Pull request merged**